tests/api2/jail.py

```diff
@@ -55,141 +56,207 @@ def test_04_fetch_bsd_release():
 
 
 def test_05_verify_bsd_release():
+    global freeze
+    freeze = False
+    global freeze_msg
+    stop_time = time.time() + 600
     while True:
-        job_status = GET(f'/core/get_jobs/?id={JOB_ID}').json()[0]
-
+        get_job = GET(f'/core/get_jobs/?id={JOB_ID}')
+        job_status = get_job.json()[0]
         if job_status['state'] in ('RUNNING', 'WAITING'):
+            if stop_time <= time.time():
+                freeze = True
+                freeze_msg = f"Failed to fetch {RELEASE}"
+                assert False, get_job.text
+                break
             time.sleep(5)
         else:
             results = POST('/jail/list_resource/', {'resource': 'RELEASE'})
-            assert RELEASE in results.json(), job_status
+            assert RELEASE in results.json(), get_job.text
             break
 
 
 def test_06_create_jail():
-    global JOB_ID
-
-    results = POST(
-        '/jail/', {
-            'release': RELEASE,
-            'uuid': JAIL_NAME,
-            'props': [
-                'bpf=yes',
-                'dhcp=on',
-                'vnet=on'
-            ]
-        }
-    )
-
-    assert results.status_code == 200, results.text
-
-    JOB_ID = results.json()
+    if freeze is True:
+        pytest.skip(freeze_msg)
+    else:
```

Comment:
Do we need all the else's in all these other files? If we say:
```
    if freeze is True:
         pytest.skip(freeze_msg)
```
Isn't that enough for it to skip? Less indented code is easier for readability.

Reply:
Yes, or they will fail.

Reply:
Basically to avoid tests that depend on the 3 Job that can fail.

Reply:
I'm saying pytest wouldn't run the rest of that block would it? Then we can skip a lot of indented code.